UI/obs-app.cpp

```diff
@@ -1098,10 +1098,75 @@ void OBSApp::ParseExtraThemeData(const char *path)
 	setPalette(pal);
 }
 
-bool OBSApp::SetTheme(std::string name, std::string path)
+bool OBSApp::ParseThemeMeta(const char *path)
 {
-	theme = name;
+	BPtr<char> data = os_quick_read_utf8_file(path);
+	CFParser cfp;
+	int ret;
+
+	cf_parser_parse(cfp, data, path);
```

Comment:
Check return value to ensure we don't crash on empty file.

Reply:
Done.